app/views/admin/event/_event_form.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= form_with(url: admin_event_modify_path) do |f| %>
```

Comment:
This is good, will be written with bootstrap_form (exists in a different branch), but good to push for now!